02-microarray/pathway-analysis_microarray_03_gsva.Rmd

```diff
@@ -347,11 +347,14 @@ mapped_df %>%
 
 We can see that the associated values vary for each row.
 
-As we mentioned earlier, we will want to remove duplicated gene identifiers in preparation for the GSVA steps later, so let's keep the Entrez IDs associated with the maximum expression value per sample.
-Note that using the maximum expression value to collapse duplicate gene identifiers is just one of the expression values that can be used here.
-See the [Gene Set Enrichment Analysis (GSEA) User guide](https://www.gsea-msigdb.org/gsea/doc/GSEAUserGuideFrame.html) for more information on other expression values that can be used [@GSEA-user-guide]. 
+As we mentioned earlier, we will want to collapse any duplicate gene identifiers in preparation for the GSVA steps later.
+GSVA is executed on a per sample basis so let's keep the maximum expression value per sample associated with the duplicate gene identifiers.
+Note that using the maximum expression value to collapse duplicate gene identifiers means preferentially choosing a value or group of a higher rank downstream.
```

Comment:
I'm not sure what this sentence means. Can explain what you mean here?

Reply:
This was based on [this comment from the draft PR](https://github.com/AlexsLemonade/refinebio-examples/pull/352#discussion_r524299264), from which I gathered that choosing the max expression value here would result in choosing the values of higher ranking in the downstream GSVA results. I can re-word this sentence for clarification.

Reply:
I think “collapsing” identifiers in general is a phrase that makes sense, but I think once we’ve stated the idea, we can be more specific about what we are doing. You use the phrase “collapse identifiers” quite a few times back to back so I’m trying to think if there more specific, clear descriptions we can replace some of those repeats with.

Reply:
This could also be a thing that I’m being weird about and the next reviewer might have a different idea or thoughts about it. 🤷‍♀️

Reply:
Let me know what you think of the latest changes in that case.